src/main/java/MysqlDatabase.java

```diff
@@ -104,6 +104,31 @@ public boolean insertCommitToDatabase(CommitStructure commit) throws SQLExceptio
         return commits;
     }
 
+    /**
+     * Selects all rows from the table "commit" in the database.
+     * @return Returns a list with all of the rows read as CommitStructure objects
+     */
```

Comment:
typo (not changed)

Reply:
https://github.com/DD2480-2021-group-22/dd2480-lab-2/pull/38/commits/c75fef29507e7f2aafc3bd951bf110086eabc62c Fixed